src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -333,17 +338,17 @@ memgraph::query::TypedValue ToTypedValue(const mgp_value &val, memgraph::utils::
       return memgraph::query::TypedValue(std::move(tv_map));
     }
     case MGP_VALUE_TYPE_VERTEX:
-      return memgraph::query::TypedValue(val.vertex_v->impl, memory);
+      return memgraph::query::TypedValue(val.vertex_v->getImpl(), memory);
```

Comment:
GetImpl, methods are in pascal case

Reply:
I didn't create the method, only reused it